### ISSUE TITLE
as.ggplot does not work perfectly

### ISSUE THREAD

**Issue by yaoluxun** [CityUHK-CompBio/HTSanalyzeR2#1]
If I run 
`p1 <- as.ggplot(~viewGSEA(res$gsca, "MSigDB_C2", topGS_GO_MF1[["MSigDB_C2"]][1]))`
`p1`
![image](https://user-images.githubusercontent.com/51792723/61896352-cb68dd80-aee2-11e9-8a18-50457735cbb0.png)

There are some texts in the middle of the plot.
If I directly run
`viewGSEA(res$gsca, "MSigDB_C2", topGS_GO_MF1[["MSigDB_C2"]][1])`
![image](https://user-images.githubusercontent.com/51792723/61896477-171b8700-aee3-11e9-9512-399198c62c09.png)
That line disappears.

**Comment by linazhu3** [comment]
Yes, that's a previous bug, just try to re-install this package and re-run.

**Comment by yaoluxun** [comment]
I think I updated it today before I ran... Anyway I’ll reinstall it and
see what happens. Thank you

**Comment by gaofeng21cn** [comment]
I re-tested this on the current code, since the plot code has been reworked since 2019.

Reproduction:

```r
library(ggplotify)
data(d7_gsca)
gs <- getTopGeneSets(d7_gsca, resultName = "GSEA.results",
                     gscs = "GO_MF", ntop = 1)[["GO_MF"]][1]

png("direct.png", width = 900, height = 700, res = 100)
viewGSEA(d7_gsca, gscName = "GO_MF", gsName = gs)
dev.off()

p <- as.ggplot(~viewGSEA(d7_gsca, gscName = "GO_MF", gsName = gs))
png("wrapped.png", width = 900, height = 700, res = 100)
print(p)
dev.off()
```

Result: both files show exactly the same plot — title, enrichment score profile, the `ES = -0.819 / FDR < 1e-02` annotation, the Positive/Negative strip and the ranked list metric. No stray text, and no missing line.

A pixel comparison at identical size gives a mean absolute difference of 0.034, with the differences confined to anti-aliased edges where the ggplot wrapper shifts the plot area by a fraction of a pixel. That is rendering jitter, not a content difference.

So the behaviour you saw is no longer reproducible; consistent with the maintainer's reply, it was a bug in the older plot code that has since been fixed. As a side note, the `ES = ...` text in the middle of the plot is intentional — it is the enrichment score annotation, and it is present in the direct call as well.

Closing as resolved. If you still see extra content, please share your `sessionInfo()` and the exact call, and we will reopen.
